en/micro-integrator/docs/references/connectors/file-connector/file-connector-example.md

```diff
@@ -0,0 +1,138 @@
+# File Connector Example
+
+The File Connector allows you to connect to different file systems and perform various operations with the file systems. The file connector uses the Apache Commons VFS I/O functionalities to execute operations.
+
+WSO2 EI File connector introduces the atomic operation related to the file system and allows you to easily manipulate files based on your requirement. The file streaming functionality using Apache Commons I/O lets you copy large files and reduces the file transfer time between two file systems resulting in a significant improvement in performance that can be utilized in file operations.
```

Comment:
```suggestion
WSO2 EI File Connector introduces the atomic operation related to the file system and allows you to easily manipulate files based on your requirement. The file streaming functionality using Apache Commons I/O allows you to copy large files and reduces the file transfer time between two file systems. This results in a significant improvement in performance that can be utilized in file operations.
```

Can we come up with a better term for "atomic operation"? It's not clear what you mean since this is normally a programming term.

Reply:
used 'independent operations' instead of 'atomic'